### ISSUE TITLE
Black screen on Android device

### ISSUE THREAD

**Issue by snuuve** [react-native-mapbox-gl/maps#995]
After upgrading to `8.0.0` i have black screen on Android device. With iOS is fine.

`adb logcat | grep Mbgl` showing this problem with connection:

```
08-18 16:22:43.793 17856 19262 I Mbgl-EGLConfigChooser: In emulator: false
08-18 16:22:43.839 17856 18559 D Mbgl-HttpRequest: [HTTP] Request with response = 404: Not Found
08-18 16:22:43.864 17856 17856 E Mbgl    : {o.bmind.cititok}[Setup]: loading style failed: HTTP status code 404
```

Versions (please complete the following information):

Platform: Android
Device: multiple
Emulator/ Simulator: device
OS: multiple
react-native-mapbox-gl 8.0.0
React Native Version 0.62.2

**Comment by ferdicus** [comment]
Maybe `accessToken` issues? 

`loading style failed: HTTP status code 404`?

**Comment by snuuve** [comment]
If i do `getAccessToken` i see it. So it's look like is assigned.

**Comment by ferdicus** [comment]
Are you using some custom style for the map?

Can you please share some code? How are you initializing the map?

**Comment by snuuve** [comment]
@ferdicus Yes, i'm using custom styles, however without this styles also doesn't work.

Acces token is set in App.ts.

Here is my cod:
```
<MapboxGL.MapView
        ref={(c) => this.map = c}
        styleURL={'mapbox://styles/lukaszmarcinkowski/ck0o0z9xb0vce1cpef8mg09ki?optimize=true'}
        style={styles.map}
        showUserLocation={true}
        regionDidChangeDebounceTime={800}
        onDidFinishLoadingMap={this._onDidFinishLoadingMap}
        onDidFinishRenderingMapFully={()=>{console.log("FULLY RENDERED MAP")}}
        attributionPosition={{top: -50, left: -80}}
        onPress={this._onMapPress}
      >
        <MapboxGL.Camera
          ref={(c) => this.camera = c}
          zoomLevel={6}
          centerCoordinate={[19.320, 52.059]}
        />
        {this._renderPostsMarkers()}
        {this.props.locationStore.getPermissionGranted() &&
        <MapboxGL.UserLocation ref={(c) => this.userLocation = c}/>}
      </MapboxGL.MapView>
`

**Comment by lukemcgregor** [comment]
This is a pretty common issue with the simulator. For me i get maybe half my tiles rendered black and other issues drawing different layers on the sim. Ive never had similar issues on a real device. see #907 

Its related to this mapbox issue https://github.com/mapbox/mapbox-gl-native/issues/14604

You can fix this in the sim by setting the renderer to SwiftShader (and rebooting the device), but it makes the app go pointlessly slow on the sim so I put it back to hardware pretty quick and learned to put up with the issue. I also use a real device pretty regularly for testing.

![image](https://user-images.githubusercontent.com/1284608/90843773-7f72bb80-e3b6-11ea-8e53-4e75b5c80711.png)

**Comment by thanmaic** [comment]
Even i have the same issue of map being blank on Android. I have created an issue as well https://github.com/react-native-mapbox-gl/maps/issues/1003 regarding the same with more details about the issue
.

**Comment by snuuve** [comment]
Ok, so if i remove `styleURL` then map is showing up. Any idea why styles doesn't work on Android?

**Comment by fjerbi** [comment]
Well, this might be a bit late, but if you still having this issue, you simply having an issue with the Api key, so make sure you follow the documentation, it happened to me and it fixed it.
